### DISCUSSION TITLE
With filter, you can't navigate between pages

### DISCUSSION THREAD

**Discussion by hugopvl** [directus/directus#8665] [Archive]
Guys,

If we have an active filter, the page browser does not work, it is always on the first page.

![image](https://user-images.githubusercontent.com/81326296/136594588-5611dbb7-805c-407d-acd0-b2c67c5e5adc.png)

![image](https://user-images.githubusercontent.com/81326296/136594526-7edd6166-ecb8-42c0-9314-9069c5088a12.png)

**Comment by rijkvanzanten** [comment]
Mind opening a new issue?

**Reply by hugopvl** [reply]
yes sorry of course
